README.md

```diff
@@ -190,6 +190,22 @@ None.
         password: similarly-secure-password
         priv: "example_db.*:ALL"
 
+## Replication Master Master
```

Comment:
There are no sections dedicated to the Master/Slave replication mode so for consistency there should be either no section for Master/Master or an equivalent section for Master/Slave.

Reply:
Plus the order of words in the title is wrong: `Master Master Replication` seems better.